content/en/hosting/requirements.md

```diff
@@ -1,49 +1,100 @@
+* [Current version](https://docs.docker.com/engine/install/) of `docker` or current version of [Docker Desktop](https://www.docker.com/products/docker-desktop/) both of which include `docker compose`. Note that the older `docker-compose` has been [deprecated in favor of Compose V2](https://www.docker.com/blog/announcing-compose-v2-general-availability/).
+
+## 3.x Docker Compose Production Hosting
+
+* 32 GB RAM  / 8 CPU / 500 GB SSD*
+* Root Access
+* Static IP with DNS entry
+* TLS certificate
+* [Current version](https://docs.docker.com/engine/install/) of `docker` which includes `docker compose`. Note that the older `docker-compose` has been [deprecated in favor of Compose V2](https://www.docker.com/blog/announcing-compose-v2-general-availability/).
 
-Hosting a CHT instance in a cloud provider like AWS or on bare-metal requires you have sufficient hardware specifications, Docker and Docker Compose installed and other infrastructure requirements met.
+_\* During some upgrades, up to 3x current space used by CouchDB can be needed_
 
-## Infrastructure Requirements
+## 4.x Kubernetes Production Hosting
 
-- A static IP
-- A DNS Entry pointing to the IP
-- TLS certificates
+This guide refers to "Kubernetes", but Medic recommends a lightweight orchestrator called [K3s](https://docs.k3s.io/) for bare-metal hosts.  The requirements below refer to K3s deployments but can be translated to other Kubernetes hosting.  For example, for cloud hosting, we recommend Amazon [Elastic Kubernetes Service](https://aws.amazon.com/eks/) (EKS) and we've also assisted in a [large k3s deployment based on VMWare]({{< relref "4.x/self-hosting/self-hosting-k3s-multinode" >}}). 
 
-## Minimum Hardware Requirements
+Be sure to see the `cht-deploy` [script](https://github.com/medic/cht-core/tree/master/scripts/deploy) that leverage the `helm` [application](https://helm.sh/docs/intro/install/).
 
-- 4 GiB RAM
-- 2 CPU/vCPU
-- 8 GB Hard Disk (SSD preferred)
-- Root Access
+* 1 x HA control-plane nodes: 2 GB RAM  / 2 CPU / 20 GB SSD
+* 3 x worker servers: 16 GB RAM  / 8 CPU / 50 GB SSD
+* 500GB storage area network (SAN)* - Will host [Persistent Volume Claims](https://kubernetes.io/docs/concepts/storage/persistent-volumes/)
+* Root Access
+* Static IP with DNS entry - Kubernetes will use this to provision a valid TLS certificate
+* `helm` [application](https://helm.sh/docs/intro/install/)
+* [K3s](https://docs.k3s.io/)
+* [Current version](https://docs.docker.com/engine/install/) of `docker` (used to bootstrap K3s)
 
-Depending on the scale of your operation these may need to be increased. Be sure to monitor disk usage so that the 8 GB can be increased as needed.
+_\* During some upgrades, up to 3x current space used by CouchDB can be needed_
 
-## Docker
 
-Install both `docker` and `docker-compose` to run CHT and related containers.
+Minimum Hardware requirements
```

Comment:
I can't make the suggested code change because it's includes more content than is in this PR, but we should delete all content between here and the "Required skills" section:

![image](https://github.com/medic/cht-docs/assets/8253488/9434d8fc-0e47-42de-b054-a9c23a46147e)

Reply:
Solved in commit [5e96555](https://github.com/medic/cht-docs/pull/1384/commits/5e96555f449e583b807ee9b8b645f43d4b53e886)